### PR TITLE
runtime-sdk: Add fee payment and disbursement

### PR DESCRIPTION
**Pull request by kostko**
TODO
* [x] Use last round committee metadata to inform fee disbursement (requires support in Oasis Core)

### CODE REVIEW

File: runtime-sdk/src/modules/accounts/mod.rs [hunk 1]
```diff
@@ -481,18 +561,102 @@ impl module::AuthHandler for Module {
...
+        if !addrs.is_empty() {
+            let amounts: Vec<_> = previous_fees
+                .iter()
+                .map(|(denom, fee)| {
+                    token::BaseUnits::new(
+                        fee.checked_div(&token::Quantity::from(addrs.len() as u64))
+                            .expect("addrs is non-empty"),
+                        denom.clone(),
+                    )
+                })
```
Comment (ptrus) [inline runtime-sdk/src/modules/accounts/mod.rs:623]: Could we also filter out zero amounts here, to avoid unnecessary adds/subs bellow?

Comment (kostko) [reply runtime-sdk/src/modules/accounts/mod.rs:623]: Good idea, will change it.

Comment (kostko) [reply runtime-sdk/src/modules/accounts/mod.rs:628]: Done.

File: runtime-sdk/src/module.rs [hunk 2]
```diff
@@ -151,6 +151,22 @@ impl MigrationHandler for Tuple {
     }
 }
 
+/// Block handler.
+#[impl_for_tuples(30)]
```
Comment (ptrus) [inline runtime-sdk/src/module.rs:155]: I see we use this on most traits that support automatic implementations (methods with no return values, no associated types). Do we actually need/use any of these in tuples anywhere (or plan to in future)? Or are we just doing this as a "convention" where we can?

Comment (kostko) [reply runtime-sdk/src/module.rs:155]: Oh yes, this is a neat feature that allows us to do things like:
https://github.com/oasisprotocol/oasis-bridge/blob/91db3f59a73611f482b86d12890ed8efde4d75b2/runtime/src/lib.rs#L14-L17

And then call all these methods on the `Runtime::Modules` type (which basically combines multiple modules).

Comment (ptrus) [reply runtime-sdk/src/module.rs:155]: Ah yeah that's nice :+1: i was actually looking for a runtime implementation but didn't found it in this repo - didn't think about looking over at bridge repo :grin:

**Review by ptrus — APPROVED** [review]

**Pull request merged**